package/obs/rmt-server.spec

```diff
@@ -59,6 +59,7 @@ BuildRequires:  sqlite-devel
 BuildRequires:  pkgconfig(systemd)
 Requires:       gpg2
 Recommends:     mariadb
+Recommends:     valkey
```

Comment:
Can this get moved to the pubcloud subpackage?

Reply:
We could, but next version we want to start working on caching on Valkey, so we'd just have to move it then.

Reply:
Ok. Will the caching be implemented in RMT core code then? We'd need to add valkey setup also to the README and the RMT helm chart.

Reply:
Planning to when we get there.

Reply:
Moved in b3428b87.